autogen.sh

```diff
@@ -1 +1,2 @@
+git submodule update --init
```

Comment:
The `.git` directory wouldn't even be included in the dist archive since it is not known by autotools. Therefore, `git` commands wouldn't work.